apps/dotcom/src/app/pages/people/PeopleData.tsx

```diff
@@ -147,7 +147,7 @@ export const people: Person[] = [
   // Shreya
   {
     name: 'Fanny Zheng',
-    position: 'Developer',
+    position: 'Developer; Co-Director of Events',
```

Comment:
:thought_balloon: in the (hopefully near) future, we'll be making separate sections for eboard and all of the teams so when that happens, I think we should just duplicate the entries (so you and Fanny will appear on the page twice) and set the positions in the appropriate sections ("Co-Director of Events" for the entry in the eboard section and "Developer" for the entry in the team section). Do you have any thoughts on this?

Reply:
Yeah, that makes sense, probably accidentally branched off the feature branch. As for the separate sections for e-board and project teams, I think that duplicating the entries would be fine. When we get to that, we can modify the position property to match this behavior.